apps/docs-snippets/src/guide/predicates/predicate-with-configurable.test.ts

```diff
@@ -28,10 +28,13 @@ describe(__filename, () => {
 
     const configurable = { WHITELISTED: newWhitelistedAddress };
     // instantiate predicate with configurable constants
-    const predicate = new Predicate(bin, wallet.provider, abi, configurable);
-
-    // set predicate data to be the same as the configurable constant
-    predicate.setData(configurable.WHITELISTED);
+    const predicate = new Predicate(
+      bin,
+      wallet.provider,
+      abi,
+      [configurable.WHITELISTED],
+      configurable
+    );
```

Comment:
Could we first set the data to a variable to better disambiguate the parameters?

```ts

    // we'll set the predicate data to be the same as the configurable constant
    const data = [configurable.WHITELISTED];

    // instantiate predicate with data and configurable constants
    const predicate = new Predicate(bin, wallet.provider, abi, data, configurable);
```